systems/plants/@RigidBodyManipulator/RigidBodyManipulator.m

```diff
@@ -1271,7 +1271,7 @@
       %                          @default 1:numel(model.body)
       if nargin < 4,          body_ids = 1:numel(model.body); end
       if nargin < 3,          robotnum = 1:numel(model.name); end
-      if all(robotnum == 0),  robotnum = 0:numel(model.name); end
+      if all(robotnum == -1),  robotnum = 0:numel(model.name); end
```

Comment:
We should probably give a name to the special frames/robotnums instead of using the actual value in many places.   This would make it easier to understand the intentions of the code without having to know that -1 is the WORLD_FRAME.